CountItems/masterProcess.js

```diff
@@ -40,17 +47,18 @@ const countMaster = new CountMaster({
         maxConcurrent: concurrentCursors,
     }),
     client: new S3UtilsMongoClient(createMongoParams(log)),
-    metrics: metricServer,
 });
 
-metricServer.onRequest((req, res) => monitoring.metricsHandler(
-    countMaster,
+const metricServer = new WebServer(8003, log).onRequest((req, res) => monitoring.metricsHandler(
     () => {
-        process.exit(1);
+        if (CountMaster.waitingForPromScraping === true) {
```

Comment:
`waitingForPromScraping` does not belong to CountMaster : it is part of the "metrics handling", which is in masterProcess.js

Reply:
done here : https://github.com/scality/s3utils/pull/312/commits/7a546979810051418f9af9bc3ed2567a0bc69eb4#diff-46cbf734fd7b9022c7bbf597a8a5deb96829b581ca817a861bbc3bf1e86274a9R81